### ISSUE TITLE
Sketchfab single result loading adds to browser session history

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#3816]
## Description

**tl;dr:** You have to press the back button twice to return to the search results after viewing a Sketchfab single result

## Details

In Firefox, the deferred loading of the Sketchfab script or iframe (TBD) pushes an extra, blank state into the browser session history. This causes the user to have to press the browser back button twice when returning to the search results. 

## Reproduction

1. Search https://openverse.org for images, filtered by the Sketchfab provider ([example](https://openverse.org/search/image?q=pokeball&source=sketchfab)
2. Select a single result
3. After the result loads, use the browser back button to return to search results
4. Observe that the button needed to be pressed twice

## Screenshots

A video of some of the behavior:

[Screencast from 2024-02-20 22-31-43.webm](https://github.com/WordPress/openverse/assets/6351754/bf913b01-0a02-48f1-9d82-0199f2fe3f8a)

## Environment

- Device: 
- OS:  Debian GNU/Linux 12 (bookworm) x86_64 
- Browser: Firefox
- Version: 115.7.0esr

## Additional context

This is most likely related to the custom approach we're taking to dynamically load the Sketchfab viewer javascript and render the iframe. Perhaps some new security feature in Firefox or other async loading functionality has caused this behavior. 

The relevant component: https://github.com/WordPress/openverse/blob/ea17972522d59a56dec2f08e8a2fa68230992dcc/frontend/src/components/VSketchFabViewer.vue

And the file for dynamic script loading: https://github.com/WordPress/openverse/blob/ea17972522d59a56dec2f08e8a2fa68230992dcc/frontend/src/components/VSketchFabViewer.vue

**Comment by szymon-polaczy** [comment]
Hi, Is still a valid issue @zackkrida? 

I tested the sketchfab images on these firefox browsers
127.0.2 (64-bit) & 115.13.0esr (64-bit)
I also didn't see this issue on brave
Version 1.67.134 Chromium: 126.0.6478.186 (Official Build) (64-bit)

I did those on Linux MInt 21.3 x86_64 so there shouldn't really be any major differences there.

If you're able to reproduce the issue could you send over the list of addons that you have in the browser that you tested? It might be that there is an issue there

Thanks

**Comment by zackkrida** [comment]
Hi @szymon-polaczy I am still seeing the issue in firefox on 128.0 (64-bit) without any addons or extensions enabled. After the sketchfab player loads, if I hit the back button once the page does not go back to the search view, rather the sketchfab player is removed, leaving a white screen like this:


https://github.com/user-attachments/assets/d85c8d57-3aa9-4eaa-a163-c5c99713ad31

**Comment by webobite** [comment]
Hi @obulat  
I would like to fix this issue Can I take this if none is working on it ?
